authorization/invitation/service/invitation_service_blackbox_test.go

```diff
@@ -551,3 +551,21 @@ func (s *invitationServiceBlackBoxTest) TestAcceptFailsForUnknownAcceptCode() {
 	_, err := s.Application.InvitationService().Accept(s.Ctx, user.IdentityID(), uuid.NewV4())
 	require.Error(s.T(), err)
 }
+
+func (s *invitationServiceBlackBoxTest) TestRescindInvitationByInvitationID() {
+	g := s.NewTestGraph()
+
+	// Create a test user - this will be the team admin
+	teamAdmin := g.CreateUser()
+
+	// Create a team
+	team := g.CreateTeam()
+
+	// Create another test user - we will invite this one to join the team
+	invitee := g.CreateUser()
+
+	inv := s.Graph.CreateInvitation(team, invitee, true)
+
+	err := s.Application.InvitationService().Rescind(s.Ctx, teamAdmin.IdentityID(), inv.Invitation().InvitationID)
+	require.NoError(s.T(), err, "Error rescinding invitation")
+}
```

Comment:
I would also use the invitation repository to try and load the invitation after it is rescinded, and ensure that it fails.